packages/dbml-parse/src/lib/interpreter/elementInterpreter/ref.ts

```diff
@@ -68,8 +69,10 @@ export class RefInterpreter implements ElementInterpreter {
 
     if (field.args[0]) {
       const settingMap = aggregateSettingList(field.args[0] as ListExpressionNode).getValue();
-      this.ref.onDelete = extractVariableFromExpression(settingMap['delete']?.at(0)?.value).unwrap_or(undefined) as any;
-      this.ref.onUpdate= extractVariableFromExpression(settingMap['update']?.at(0)?.value).unwrap_or(undefined) as any;
+      const deleteSetting = settingMap['delete']?.at(0)?.value;
+      this.ref.onDelete = deleteSetting instanceof IdentiferStreamNode ? extractStringFromIdentifierStream(deleteSetting).unwrap_or(undefined) : extractVariableFromExpression(deleteSetting).unwrap_or(undefined) as any
+      const updateSetting = settingMap['update']?.at(0)?.value;
+      this.ref.onUpdate = updateSetting instanceof IdentiferStreamNode ? extractStringFromIdentifierStream(updateSetting).unwrap_or(undefined) : extractVariableFromExpression(updateSetting).unwrap_or(undefined) as any
```

Comment:
My suggestion:
* fix code style
* I think the correct type for onUpdate and onDelete should be string. please also update that `type.ts`
```suggestion
      this.ref.onDelete = deleteSetting instanceof IdentiferStreamNode
        ? extractStringFromIdentifierStream(deleteSetting).unwrap_or(undefined) 
        : extractVariableFromExpression(deleteSetting).unwrap_or(undefined) as string;
      const updateSetting = settingMap['update']?.at(0)?.value;
      this.ref.onUpdate = updateSetting instanceof IdentiferStreamNode
        ? extractStringFromIdentifierStream(updateSetting).unwrap_or(undefined)
        : extractVariableFromExpression(updateSetting).unwrap_or(undefined) as string;
```